.syncpackrc

```diff
@@ -238,6 +238,15 @@
 			],
 			"pinVersion": "~29.1.1"
 		},
+		{
+			"dependencies": [
+				"wireit"
+			],
+			"packages": [
+				"**"
+			],
+			"pinVersion": "^0.14.1"
```

Comment:
For something as crucial as wireit, and for consistency, I wonder if we should just use exact version? Especially since wireit isn't at 1.0 yet an increment of 14 to 15 could mean some relatively significant or more risky change?

Reply:
Good call, I've done that.